aries-site/src/examples/templates/FilterControls/ResultsSummary.js

```diff
@@ -3,24 +3,36 @@ import { Box, Text } from 'grommet';
 import { useFilters } from '.';
 
 export const ResultsSummary = () => {
-  const { data, filteredResults, isFiltered } = useFilters();
+  const { data, filteredResults, isFiltered, selected } = useFilters();
+  const textSize = 'small';
 
-  return (
+  return selected.length ? (
     <Box direction="row" gap="xxsmall">
-      <Text size="small" weight="bold">
+      <Text size={textSize} weight="bold">
+        {selected.length}
+      </Text>
+      <Text size={textSize}>of</Text>
+      <Text size={textSize} weight="bold">
+        {isFiltered ? filteredResults.length : data.length}
+      </Text>
+      <Text size={textSize}>items selected</Text>
```

Comment:
Would this be even more clear if this read `3 of 24 filtered results selected`? Or does it get more confusing?

```suggestion
      <Text size={textSize}>filtered results selected</Text>
```

Reply:
I think it gets a bit more confusing. Maybe just `3 of 24 results selected` could work -- for now, I've just aligned directly with Diane and was planning to leave further discussions on this patter to a separate chat (since I know there were some other ideas explored as well by Raja/Vicky potentially)